client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/style.scss

```
@@ -13,7 +13,7 @@
 		}
 		.woocommerce-admin__business-details__selective-extensions-bundle__extension {
 			display: flex;
-			padding: $gap-large 0 $gap-large 30px;
+			padding: $gap-large 20px $gap-large 30px;
```

Shouldn't this be `30px` so that left and right have equal padding?

```suggestion
			padding: $gap-large 30px;
```

---

Nice catch! I changed it in the commit c71b619